server/requirements.txt

```diff
@@ -1,19 +1,23 @@
+atomicwrites==1.2.1
+attrs==18.2.0
 certifi==2018.4.16
 chardet==3.0.4
 click==6.7
+colorama==0.4.0
 defusedxml==0.5.0
+docopt==0.6.2
 Flask==1.0.2
-fuzzywuzzy[speedup]==0.17.0
-idna==2.6
+fuzzywuzzy==0.17.0
```

Comment:
The `fuzzywuzzy[speedup]` included an extra module that we want. I'm guessing you did a `pip freeze` here?

Reply:
yes, I did, as mentioned above. The speedup was not in my freeze once things worked. I presumed it was a mistake or a change in dependency over time. Meaning, that maybe the pip resources changed at the source.